src/Workspaces/CoreTest/ExtensionsTests/DocumentExtensionsTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests.ExtensionsTests
+{
+    public class DocumentExtensionsTests
+    {
+        private readonly AdhocWorkspace _ws = new AdhocWorkspace();
+        private readonly Project _emptyProject;
+
+        public DocumentExtensionsTests()
+        {
+            _emptyProject = _ws.AddProject(
```

Comment:
Thanks, @Pilchie, but the `TestWorkspace` in the `ServicesTest` project doesn't seem to do that. There's one in `EditorServicesTest`, but I'm assuming referencing it is not an option. I copied this from another test in the same project.